resources/views/bread/browse.blade.php

```diff
@@ -54,10 +54,14 @@
                                                 @if ($data->{$row->field} && isset($options->relationship))
                                                     {{ $data->{$row->field}->implode($options->relationship->label, ', ') }}
                                                 @endif
+                                            @elseif($row->type == 'select_dropdown' && property_exists($options, 'options'))
+
+                                                <?php echo $options->options->{$data->{$row->field}};?>
```

Comment:
What exactly does this line do?

Also please use `{{ ... }}` instead of `<?php echo ... ; ?>`